frontend/src/__tests__/unit/useFetchState.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { act } from '@testing-library/react';
+import useFetchState from '~/utilities/useFetchState';
+import { expectHook, testHook } from './testUtils/hooks';
+
+jest.useFakeTimers();
+
+describe('useFetchState', () => {
+  it('should be successful', async () => {
+    const renderResult = testHook(
+      useFetchState,
+      () => Promise.resolve('success-test-state'),
+      'default-test-state',
+    );
+
+    expectHook(renderResult)
+      .toStrictEqual(['default-test-state', false, undefined, expect.any(Function)])
```

Comment:
(we already talked about this, for posterity) Can we make a utility function for tests to not have to repeat the 4 variables of `useFetchState` -- something like `standardUseFetchState(...up to 4 values)`

eg
```suggestion
      .toStrictEqual(standardUseFetchState('default-test-state'))
```